libctru/include/3ds/services/y2r.h

```diff
@@ -1,113 +1,361 @@
+ * @param image_size The total size of the data buffer.
+ * @param transfer_unit Specifies the size of 1 DMA transfer. Usually set to 1 line. This has to be a divisor of image_size.
+ * @param transfer_unit Specifies the gap (offset) to be added after each transfer. Can be used to convert only a part of an image.
+ *
+ * This specifies the Y data buffer for the planar input formats (INPUT_YUV42*_INDIV_*).
+ * The actual transfer will only happen after calling @ref Y2RU_StartConversion.
+ */
 Result Y2RU_SetSendingY(const void* src_buf, u32 image_size, u16 transfer_unit, u16 transfer_gap);
+
+/**
+ * @brief Configures the U plane buffer
+ * @param src_buf A pointer to the beginning of your Y data buffer.
+ * @param image_size The total size of the data buffer.
+ * @param transfer_unit Specifies the size of 1 DMA transfer. Usually set to 1 line. This has to be a divisor of image_size.
+ * @param transfer_unit Specifies the gap (offset) to be added after each transfer. Can be used to convert only a part of an image.
+ *
+ * This specifies the U data buffer for the planar input formats (INPUT_YUV42*_INDIV_*).
+ * The actual transfer will only happen after calling @ref Y2RU_StartConversion.
+ */
 Result Y2RU_SetSendingU(const void* src_buf, u32 image_size, u16 transfer_unit, u16 transfer_gap);
+
+/**
+ * @brief Configures the V plane buffer
+ * @param src_buf A pointer to the beginning of your Y data buffer.
+ * @param image_size The total size of the data buffer.
+ * @param transfer_unit Specifies the size of 1 DMA transfer. Usually set to 1 line. This has to be a divisor of image_size.
+ * @param transfer_unit Specifies the gap (offset) to be added after each transfer. Can be used to convert only a part of an image.
+ *
+ * This specifies the V data buffer for the planar input formats (INPUT_YUV42*_INDIV_*).
+ * The actual transfer will only happen after calling @ref Y2RU_StartConversion.
+ */
 Result Y2RU_SetSendingV(const void* src_buf, u32 image_size, u16 transfer_unit, u16 transfer_gap);
+
+/**
+ * @brief Configures the YUYV source buffer
+ * @param src_buf A pointer to the beginning of your Y data buffer.
+ * @param image_size The total size of the data buffer.
+ * @param transfer_unit Specifies the size of 1 DMA transfer. Usually set to 1 line. This has to be a divisor of image_size.
+ * @param transfer_unit Specifies the gap (offset) to be added after each transfer. Can be used to convert only a part of an image.
+ *
+ * This specifies the YUYV data buffer for the packed input format @ref INPUT_YUV422_BATCH.
+ * The actual transfer will only happen after calling @ref Y2RU_StartConversion.
+ */
 Result Y2RU_SetSendingYUYV(const void* src_buf, u32 image_size, u16 transfer_unit, u16 transfer_gap);
-Result Y2RU_IsDoneSendingYUYV(bool* is_done);
+
+/**
+ * @brief Configures the destination buffer
+ * @param src_buf A pointer to the beginning of your destination buffer. It can't be in VRAM.
```

Comment:
I'd say "destination buffer in FCRAM." instead of the note at the end. Also mention that the buffer does not need to be allocated in the linear heap.